tasks/main.yml

```diff
@@ -56,16 +55,10 @@
         path: /var/log/clamav
         state: directory
 
-    # This happens automatically on Debian when we start the
-    # clamav-freshclam service
+    # This happens automatically on Debian and Ubuntu when the
+    # clamav-freshclam service starts on the next boot
     - name: Run freshclam (Fedora)
       command: /usr/bin/freshclam
       args:
         creates: /var/lib/clamav/bytecode.cvd
   when: ansible_facts['distribution'] == 'Fedora'
-
-- name: Wait for new signatures to be downloaded and installed by freshclam
```

Comment:
I know this is perturbing the build... there are a couple reasons I want this completed before the first boot.
 - The AV report generator will be stating these file to get their age.
 - Not all instances have internet access.  It would be nice if they had some signatures even if they are a few weeks old.

Reply:
OK, I will revert the commits that removed that stuff.

Reply:
See 1182af8, 13287eb, and 909c548.

Reply:
Thanks @jsf9k